source/Random.h

```diff
@@ -39,8 +40,10 @@ class Random {
 	static uint32_t Polya(uint32_t k, double p = .5);
 	// Get a number from a binomial distribution (i.e. integer bell curve).
 	static uint32_t Binomial(uint32_t t, double p = .5);
-	// Get a normally distributed number (mean = 0, sigma= 1).
-	static double Normal();
+	// Get a normally distributed number (mean = 0, sigma= 1) using std::normal_distribution.
+	static double StdNormal();
+	// Get a normally distributed number (mean = 0, sigma = 1) using the Box-Muller transform.
+	static double BMNormal(double mean, double sigma);
```

Comment:
I didn't remove the std one because it was already there.  I thought I had implied in my comments along the way that I wanted to get rid of it but it didn't concern me (my PR) directly so I didn't.

As far as the name I wanted to keep it consistent but I suppose Box-Muller is only for normal so the name you suggested could work too.

Reply:
Addressed in e069e0c0e24647e8fbb82cc51e21a0b831d215a7.